src/components/AppProvider/AppProvider.tsx

```diff
@@ -41,6 +42,7 @@ export interface AppProviderProps extends AppBridgeOptions {
   features?: Features;
   /** Inner content of the application */
   children?: React.ReactNode;
+  UNSTABLE_telemetry?: TelemetryObject; // eslint-disable-line babel/camelcase
```

Comment:
We tend to avoid the trailing comment syntax and instead use `eslint-disable-next-line`